service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/support/MigrateCaseController.java

```diff
@@ -72,6 +76,22 @@ private void run3214(CaseDetails caseDetails) {
         }
     }
 
+    private void run3239(CaseDetails caseDetails) {
+        CaseData caseData = getCaseData(caseDetails);
+        validateFamilyManNumber("FPLA-3239", caseData.getFamilyManCaseNumber(), caseData);
```

Comment:
oops, should be checking hardcoded family man number (but the migration tool checks this anyway)